src/modules/layout/components/ToolbarItem.js

```diff
@@ -20,7 +20,7 @@ class ToolbarItem extends React.PureComponent<PropsType> {
   render () {
     const { href, text, icon } = this.props
     return (
-      <StyledToolbarItem href={href} aria-label={text}>
+      <StyledToolbarItem href={href} title={text}>
```

Comment:
`aria-label` was better than `title`. Title doesn't make sense here. (We already have a custom tooltip for that)